.vscode/launch.json

```diff
@@ -0,0 +1,17 @@
+{
+    // VSCode launch configuration settings file for mqttwarn.
```

Comment:
I shall assume comments are permitted in this file specifically for VScode.

Reply:
That was new to me as well, but right - nothing croaks on this end. It is probably already using JSON5?